css-positioning.tar/css-positioning/fixed.css

```diff
@@ -0,0 +1,13 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+
+  /* your code here */
+  align-items: center;
+  position: fixed;
+  left:750px;
```

Comment:
another thing you can do is set `left: 0` and `right: 0` to make sure the `.page-title` takes up the entire width of the body